tests/cypress/e2e/jcontent/breadcrumbsPageBuilder.cy.ts

```diff
@@ -0,0 +1,31 @@
+import {JContent, JContentPageBuilder} from '../../page-object';
+
+describe('Breadcrumbs in page builder', () => {
```

Comment:
Can we specify it's boxes breadcrumbs in the label as there is already tests for breadcrumbs